README.md

```diff
@@ -1,56 +1,90 @@
+~ $ mkdir my-git-project //Creates new directory
+~ $ cd my-git-project //Moves into the newly created directory
 ```
 
-After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/next-big-thing/.git/`. Git is letting you know that it created a new repository within the hidden `.git` folder in `next-big-thing`. This hidden directory, `.git`, is what Git uses to keep track of all the different versions of your code.
+The files are available now, so how do we get them in the cloud so we can collaborate with 
+other developers or backup your progress thus far? 
+
+## State initializing Git repository with `git init`
+
+Make sure you are in the directory where you want to initiliaze `git` on your computer. If 
+you would like control, you can use a few commands to get things up and running quickly. In 
+your terminal type:
+
+```
+my-git-project $ git init
+Initialized empty Git repository in /Users/avi/my-git-project/.git/
+```
 
-You don't need to do anything with the `.git` directory –– it'll remain hidden. Just remember that if a directory has a `.git` directory inside of it, it is controlled by Git. It's also nice to know there's no magic behind the process: if Git is keeping track of multiple versions of files that you can't see, Git has to be putting them somewhere inside your computer. That somewhere is `.git`.
+After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/my-git-project/.git/`. Git is letting you know that it created a new repository 
+within the hidden `.git` folder in `my-git-project`. This hidden directory, `.git`, is what Git uses 
+to keep track of all the different versions of your code.
 
-At this point, though, there's nothing for Git to keep track of. We'll have to create some files and folders in our project first.
+You won't need to do anything with the `.git` directory –– it'll remain hidden. Just remember that if 
+a directory has a `.git` directory inside of it, it is controlled by Git. It's also nice to know there's 
+no magic behind the process: if Git is keeping track of multiple versions of files that you can't see, 
+Git has to be putting them somewhere inside your computer. That somewhere is `.git`.
 
-## Checking the status of your repository with `git status`
+## Use `git status` to check the status of your repository 
 
-Let's start our project out by creating a `README.md` that describes the project. Make your new file by typing `touch README.md` from within the `next-big-thing` directory. You won't see any output after `touch`, but you will see a new file has been created if you type `ls`.
+Let's create a `README.md` that describes the project. Make your new file by typing `touch README.md` 
+from within the `my-git-project` directory. You won't see any output after `touch`, but you will see a 
+new file has been created if you type `ls`, which gives a list of all the files in the directory.
```

Comment:
Maybe re-use `ls` in the context of "hidden `.git` directory above. If you're going to teach the unix commands, make sure you share the technique of "`ls` to validate changes"